### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by boaz-codota**

### CODE REVIEW

File: .github/workflows/continues_integration.yml [hunk 1]
```diff
@@ -21,4 +21,25 @@ jobs:
...
+        - name: Use Node.js 10.x
+          uses: actions/setup-node@v1
+          with:
+            node-version: 10.x
+        - name: Install
+          run: npm ci --no-optional
+        - run: xvfb-run -a npm test
+          if: runner.os == 'Linux'
+        - run: npm test
+          if: runner.os != 'Linux'
```
Comment (dimacodota) [inline .github/workflows/continues_integration.yml:44]: why?

**Review by dimacodota — APPROVED** [review]
LGTM, well done 👍

File: .github/workflows/continues_integration.yml (hunk 1 above)
Comment (boaz-codota) [reply .github/workflows/continues_integration.yml:44]: @dimacodota From their manual: https://code.visualstudio.com/api/working-with-extensions/continuous-integration#github-actions

Comment (dimacodota) [reply .github/workflows/continues_integration.yml:44]: ok cool, I've missed the `xvfb-run -a npm test`

**Pull request merged**